src/main/java/hu/oe/nik/szfmv/automatedcar/model/AutomatedCar.java

```diff
@@ -1,12 +1,13 @@
-package hu.oe.nik.szfmv.automatedcar;
+package hu.oe.nik.szfmv.automatedcar.model;
 
-import hu.oe.nik.szfmv.automatedcar.model.WorldObject;
+import hu.oe.nik.szfmv.automatedcar.model.interfaces.ICrashable;
+import hu.oe.nik.szfmv.automatedcar.model.interfaces.IDynamic;
 import hu.oe.nik.szfmv.automatedcar.systemcomponents.Driver;
 import hu.oe.nik.szfmv.automatedcar.virtualfunctionbus.VirtualFunctionBus;
 import org.apache.logging.log4j.LogManager;
 import org.apache.logging.log4j.Logger;
 
-public class AutomatedCar extends WorldObject {
+public class AutomatedCar extends WorldObject implements ICrashable, IDynamic {
```

Comment:
hozzunk létre egy Car-t ami extends WorldObject implements ICrashable, IDynamic

ez pedig: public class AutomatedCar extends Car

Reply:
Köszönöm, én is így akartam, hiszed vagy sem :D